content/en/docs/attestation/policies/_index.md

```diff
@@ -43,6 +43,17 @@ The input to the resource policies is the URI of the resource that is being requ
 attestation token which was generated by attestation service for the guest making the request.
 
 ### Basic Policies
+The KBS Client has a few simple policies built into it.
```

Comment:
these are available if the staging images are used but what about the trustee-operator based installation?

Reply:
The [instructions](https://confidentialcontainers.org/docs/attestation/client-tool/) we have for the KBS client pull the latest one. With the operator I think there is a container with the client in it. We don't mention this anywhere in the docs, but maybe that's future work. That container will pick up these changes at the release presumably.

Reply:
you can scratch my question, my brain made me think these are the built-in AS policies :-)